code/modules/mob/living/simple_animal/shade.dm

```diff
@@ -94,6 +94,9 @@
 		return 0 //under effects of time magick
 	. = ..()
 
+	regular_hud_updates()
+
+/mob/living/simple_animal/shade/regular_hud_updates()
 	update_pull_icon() //why is this here?
```

Comment:
Could you remove this comment? Coders should be baffled no more by the pull hud update proc being inside the general hud update proc.